src/python/gudhi/point_cloud/dtm.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, k, q=2, **kwargs):
         Args:
             k (int): number of neighbors (possibly including the point itself).
             q (float): order used to compute the distance to measure. Defaults to 2.
-            kwargs: same parameters as :class:`~gudhi.point_cloud.knn.KNearestNeighbors`, except that
+            kwargs: same parameters as :ref:`K nearest neighbors`, except that
```

Comment:
I think the link to the class was more precise (could even link to `__init__`), it is really "same arguments as this function". We could have more than just this one class in the section, and then the link wouldn't be so clear.

Reply:
I did it this, because when you click on the link, you will see the `requires` field.

Reply:
Hmm, I think the priority is for the user to see what the arguments are, not the requires thing (and the argument description kind of mentions the dependencies already), but if you prefer your version, ok.

Reply:
Ok, I rolled it back